src/main/kotlin/com/swisscom/health/des/cdr/clientvm/config/CdrClientContext.kt

```diff
@@ -77,4 +88,36 @@ class CdrClientContext {
             }
         }
 
+    @Bean
+    @Profile("dev", "test")
```

Comment:
If you use the profile expression `dev | test` here and then `!(dev | test)` for the confidentialClientApp bean, then it would be more obvious that those two beans are mutually exclusive.